Simple Incremental/Assets/Scripts/EnemyManager.cs

```diff
@@ -21,13 +28,61 @@ private void Awake()
         }
     }
 
-    public void RemoveEnemy(GameObject go)
+    private void Start()
     {
-        enemies.Remove(go.GetComponent<CharacterHealth>());
+        StartCoroutine(SpawnEnemiesControl());
     }
 
     public void AddEnemy(GameObject go)
     {
         enemies.Add(go.GetComponent<CharacterHealth>());
+    } 	
+    public void SpawnEnemy()
+    {
+        //Spawn a new enemy at a random location 
+        Vector3 randomRange;
```

Comment:
You are spawning enemies space invader-style and always on the edge of the screen. Enemies should be spawning outside of the screen completely to avoid seemingly "blipping" into existence.

Reply:
I can adjust the values outside the screen by just increasing the configurable parameters. I had a few different idea about spawning and didn't know what we wanted this seemed like the easiest for now. A few of my ideas were to have a list of spawn points that could be randomly selected from. Another idea was to have spawn areas which an enemy can be spawned anywhere in the box. Based on everyones feedback so far i think ill make a game object that has a spawn script which will allow configuring a spawn distance. So if someone wants multiple spawn areas they can just drop multiple in and configure spawn time and distance. Does that sound ok?

Reply:
What is the purpose of the spawn distance? So enemies can spawn within a range of that gameObject? You still have random-ness and imprecision. Enemies spawning randomly isn't critical to the functionality and can be added at a later time. Consider just spawning enemies directly on the gameObject's position and remove the variance factor for now (can comment, or move to a separate method and not use it yet)